geode-lucene/src/upgradeTest/java/org/apache/geode/cache/lucene/LuceneSearchWithRollingUpgradeDUnit.java

```diff
@@ -142,23 +142,17 @@ void executeLuceneQueryWithServerRollOvers(String regionType, String startingVer
 
       server1 = rollServerToCurrentCreateLuceneIndexAndCreateRegion(server1, regionType,
           testingDirs[0], shortcutName, regionName, locatorPorts, reindex);
-      verifyLuceneQueryResultInEachVM(regionName, expectedRegionSize, server1);
       expectedRegionSize += 5;
-      putSerializableObjectAndVerifyLuceneQueryResult(server1, regionName, expectedRegionSize, 5,
-          15, server2, server3);
+      putSerializableObject(server1, regionName, 5, 15);
```

Comment:
My review is mostly for the gradle and pom changes involved in rolling the Lucene dependency version. Seems fine to roll a major-version of our dependency in a minor version of Geode. 

My question here, is why are we not validating the result of this `put` ?

Reply:
Between Lucene version happened changes on Lucene format, so it cannot read with old server new indexes. So we agree on the dev list to queue puts until all members are upgraded. You can find it in the description of this PR.